sgkit/stats/preprocessing.py

```diff
@@ -185,3 +185,91 @@ def filter_partial_calls(
     )
     new_ds[variables.call_genotype_complete].attrs["mixed_ploidy"] = mixed_ploidy
     return conditional_merge_datasets(ds, new_ds, merge)
+
+
+def mean_impute(
+    ds: Dataset,
+    variable: str,
+    dim: Union[Hashable, Sequence[Hashable]] = "samples",
+    merge: bool = True,
+) -> Dataset:
+    """Mean impute a masked variable
```

Comment:
It would be helpful to give a more descriptive follow up sentence here, like say

> This replaces missing data for the specified variable with the mean of the non-missing values.